cruise.umple/test/cruise/umple/implementation/ClassTemplateTest_InternalConstant.ump

```diff
@@ -0,0 +1,3 @@
+class Student{
+    private static final String Name = "XYZ";
```

Comment:
I thought you had changed these to internal const. what is this file all about?

Reply:
This was a mistake, my editor has auto-indented the code and I had not realised that the file was staged for commit. The file is for code generation test for the internal feature I had previously worked on